src/test/java/edu/harvard/iq/dataverse/mydata/SolrQueryFormatterTest.java

```diff
@@ -127,12 +133,103 @@ private void msgt(String s){
+                { new ArrayList<Long>(),                        "",          "dvObjectType", "sliceOfIds must have at least 1 value",           IllegalStateException.class },
+                { new ArrayList<Long>(),                        "paramName", null,           "sliceOfIds must have at least 1 value",           IllegalStateException.class },
+                { new ArrayList<Long>(),                        "paramName", "",             "sliceOfIds must have at least 1 value",           IllegalStateException.class },
+                { new ArrayList<Long>(),                        "paramName", "dvObjectType", "sliceOfIds must have at least 1 value",           IllegalStateException.class },
+
+                { new ArrayList<Long>() {{ add(null); }},       null,        null,           "paramName cannot be null",                        NullPointerException.class },
+                { new ArrayList<Long>() {{ add(null); }},       null,        "",             "paramName cannot be null",                        NullPointerException.class },
+                { new ArrayList<Long>() {{ add(null); }},       null,        "dvObjectType", "paramName cannot be null",                        NullPointerException.class },
+                { new ArrayList<Long>() {{ add(null); }},       "",          null,           "(:())",                                           null },
+                { new ArrayList<Long>() {{ add(null); }},       "",          "",             "(:() AND dvObjectType:())",                       null },
+                { new ArrayList<Long>() {{ add(null); }},       "",          "dvObjectType", "(:() AND dvObjectType:(dvObjectType))",           null },
+                { new ArrayList<Long>() {{ add(null); }},       "paramName", null,           "(paramName:())",                                  null },
+                { new ArrayList<Long>() {{ add(null); }},       "paramName", "",             "(paramName:() AND dvObjectType:())",              null },
+                { new ArrayList<Long>() {{ add(null); }},       "paramName", "dvObjectType", "(paramName:() AND dvObjectType:(dvObjectType))",  null },
+
+                { new ArrayList<Long>(Arrays.asList(1L)),       null,        null,           "paramName cannot be null",                        NullPointerException.class },
+                { new ArrayList<Long>(Arrays.asList(1L)),       null,        "",             "paramName cannot be null",                        NullPointerException.class },
+                { new ArrayList<Long>(Arrays.asList(1L)),       null,        "dvObjectType", "paramName cannot be null",                        NullPointerException.class },
+                { new ArrayList<Long>(Arrays.asList(1L)),       "",          null,           "(:(1))",                                          null },
+                { new ArrayList<Long>(Arrays.asList(1L)),       "",          "",             "(:(1) AND dvObjectType:())",                      null },
```

Comment:
The expected query part seems not to make sense.
=> Check whether `dvObjectType` is empty or not and throw an exception if it is empty.